docs/ruleset_functions/G311_exceptions/zone_meets_G3_1_1c

```diff
@@ -0,0 +1,74 @@
+# zone_meets_G3_1_1c
+
+**Schema Version:** 0.0.21
+
+**Description:** determines whether a given zone meets the G3_1_1c exception "If the baseline HVAC system type is 5, 6, 7, 8, 9, 10, 11, 12, or 13 use separate single-zone systems conforming with the requirements of system 3 or system 4 (depending on building heating source) for any spaces that have occupancy or process loads or schedules that differ significantly from the rest of the building. Peak thermal loads that differ by 10 Btu/h·ft2 (2.930710 W/sf) or more from the average of other spaces served by the system, or schedules that differ by more than 40 equivalent full-load hours per week from other spaces served by the system, are considered to differ significantly. Examples where this exception may be applicable include but are not limited to natatoriums and continually occupied security areas. This exception does not apply to computer rooms."
+"average" is not particularly well defined in this exception. 
+for the peak W/sf calculation I've assumed:
+	1. the zone being tested is not included
+ 	2. the average: (each space coincident peak W) / total_area
+
+**Inputs:** 
+- **RMR**
+- **zone**
+
+**Returns:**  
+- **result**: a string - either "C" or "No"
+ 
+**Function Call:**
+- **get_baseline_system_types**
+- **get_zone_peak_internal_load**
+
+## Logic:
+- set the result variable to "No" - only a positive test can give it a different value: `result = "No"`
+- get the building_segment the zone is in:
+`for building_segmentin RMR.building.buildings:`
+	- check if the zone is in the building_segment:
+	`if zone in building_segment.zones:`
+		- break - this means the variable "building_segment" is the building_segment that contains the zone:
+		`break`
```

Comment:
There seems to be an existing function that you can use https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_list_hvac_systems_associated_with_zone.md. Seems like we need the following logic:
1. Call get_list_hvac_systems_associated_with_zone. This will return all HVAC systems associated with the zone.  Since in the baseline we expect to have one system serving each zone, there should only be one system in the list but perhaps we can structure this function to also work for scenarios where there is more than one system.
2. For each system in the list,
      - check if it is multizone (5, 5b, 7, 7a, etc.) via get_baseline_system_types(RMR). If yes,
      - get_dict_of_zones_and_terminal_units_served_by_hvac_sys (or perhaps get_hvac_zone_list_w_area would work better) to get all other zones served by this HVAC system
      - see if the test zone meets either exception criteria

Reply:
#1 - awesome.  Much faster
#2 - this line: `segment_system_types_b = {key: system_types_b[0] for key in system_types_b if(len(set(system_types_b[key])&set(relevant_systems)) > 0)` does some fancy code to do the first part

the second part - yes, good points.  I've updated the code